common/models/user.js

```diff
@@ -69,11 +69,23 @@ module.exports = function(User) {
    * customize how access tokens are generated
    *
    * @param {Number} ttl The requested ttl
+   * @param {Object} options The options for access token
```

Comment:
You should also update https://github.com/strongloop/loopback/pull/1152/files#diff-5d81af95449574544e7317f3eda2badeL144 with some docs for this.